extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -93,26 +92,48 @@ export class SubscriptionService {
     this.subscriptionPlatforms_.push(subscriptionPlatform);
 
     subscriptionPlatform.getEntitlements()
-        .then(() => this.processEntitlement_());
+        .then(entitlements => this.resolveEntitlementsToStore_(entitlements));
   }
 
-  /** @private */
-  processEntitlement_() {
-    // TODO(@prateekbh): process and unblock marup here.
+  /**
+   * @param {?Entitlements} entitlements
+   * @private
+   */
+  processEntitlements_(entitlements) {
+    if (entitlements === null) {
+      // TODO(@prateekbh): Show UI that no eligible entitlement found
+      return;
+    }
+
+    this.renderer_.toggleLoading(false);
+  }
+
+  /**
+   * @param {!Entitlements} entitlements
+   * @private
+   */
+  resolveEntitlementsToStore_(entitlements) {
+    this.entitlementStore_.resolveEntitlement(entitlements.service,
+        entitlements);
   }
 
   /** @private */
   start_() {
     this.initialize_().then(() => {
-      // TODO(@prateekbh): Start and stop loading indicator. See
-      // `Renderer.toggleLoading`.
+      this.renderer_.toggleLoading(true);
       // TODO(@prateekbh): Read the service ids in EntitlementStore constructor
       // from page config.
-      this.entitlementStore_ = new EntitlementStore(['foo', 'bar']);
+      this.entitlementStore_ =
```

Comment:
Well, you already came up with the notion of `platformConfig` above. So at this point, let's store it and use it here.

Reply:
Will read this from config in next PR